docs/result.md

```diff
@@ -0,0 +1,12 @@
+#Return_result
+
+a POST request is invoked by the client to the server. It returns a JSON file with information like client_id, job, data, and jobs.
+
+## JSON definition
+
+A job is communicated as a simple JSON object where key names and values are lower case:
+
+* `client_id`: a unique identifier of the client returning the result.
+* `job`: work that needs to be done by the client.
```

Comment:
Sounds like it isn't really differentiated from jobs. The job field is the current job's information associated with the data being returned. Whereas the way it is worded, it sounds like it is just more work yet to be done

Reply:
Maybe even get a bit more technical and specific. For example: Job is a job object, Jobs is a list of job objects, and data is a JSON object